istio-operator/istio-operator.spec

```diff
@@ -55,15 +62,24 @@ Istio-operator is a kubernetes operator to manage the lifecycle of Istio.
 %prep
 
 rm -rf OPERATOR
+
 mkdir -p OPERATOR/src/github.com/maistra/istio-operator
 tar zxf %{SOURCE0} -C OPERATOR/src/github.com/maistra/istio-operator --strip=1
 
+mkdir -p OPERATOR/src/github.com/maistra/istio
+tar zxf %{SOURCE1} -C OPERATOR/src/github.com/maistra/istio --strip=1
+
 %build
 cd OPERATOR
 export GOPATH=$(pwd):%{gopath}
 cd src/github.com/maistra/istio-operator/tmp/build/
 ./build.sh
 
+cd OPERATOR
+cp -r src/github.com/maistra/istio/install/kubernetes/helm/ src/github.com/maistra/istio-operator/tmp/build/_output
+cd src/github.com/maistra/istio-operator/
+MAISTRA_VERSION=0.10.0 SOURCE_DIR=. HELM_DIR=src/github.com/maistra/istio-operator/tmp/build/_output/helm ./tmp/build/patch-charts.sh
```

Comment:
Can you pick up MAISTRA_VERSION from the Version variable (line 30)?

Reply:
updated.  please check.  i'm assuming that it should be `%{VERSION}`.

Reply:
Thanks